script.js

```diff
@@ -187,6 +189,47 @@ function prb_plus()
   var x = document.getElementById('prb_count').innerHTML.split(": ");
   var cnt=Number(x[x.length-1])+1;
   document.getElementById('prb_count').innerHTML="Problems Count: "+cnt.toString();
+  updateAverage(cnt);        
```

Comment:
Add a check here so that it only updates avg when the record is ON. Like, lets say record is off and the user tries to do a +1 then it should not do anything right.